src/jetstream/plugins/desktop/kubernetes/tokens.go

```diff
@@ -14,6 +14,10 @@ func (d *TokenStore) FindAuthToken(userGUID string, encryptionKey []byte) (inter
 	return d.store.FindAuthToken(userGUID, encryptionKey)
 }
 
+func (d *TokenStore) ListAuthToken(encryptionKey []byte) ([]interfaces.TokenRecord, error) {
+	return d.store.ListAuthToken(encryptionKey)
```

Comment:
"This was used by a previous implementation, but i forgot to also remove the function in the TokenRepository. I will remove it since it's not used anymore."